README.md

```diff
@@ -26,6 +26,28 @@ under that directory will contain instructions on how to run the sample.
 
 If desired, you can easily use the Maven Archetype from a Maven capable IDE such as [Eclipse](https://www.eclipse.org/ide). The generated starter code is simply Maven projects. You can easily load, explore and run the code in a Maven capable IDE such as [Eclipse](https://www.eclipse.org/ide).
 
+## Instructions On How To Use Archetypes From An IDE Such As Eclipse
+
+# If you are using an enterprise version of Eclipse for a Jakarta EE project, you can follow these steps to create a new Maven project with an archetype:
+
+Open Eclipse and select "File > New > Other..." from the menu bar.
+In the "New" dialog, expand the "Maven" folder and select "Maven Project".
+Click "Next".
+In the "New Maven Project" dialog, select "Create a simple project".
+Click "Next".
+In the "New Maven Project" dialog, enter the Group Id, Artifact Id, and Version for your project.
+Click "Next".
+In the "Select an Archetype" dialog, click the "Add Archetype" button.
```

Comment:
Why is "Add Archetype" needed here? Our archetype will already be in Maven Central or installed as a SNAPSHOT in the local repository.